source/styles/layout/_hero-block.scss

```diff
@@ -0,0 +1,44 @@
+.hero-block {
+	background: url('../images/inwestycja_banner.png') no-repeat;
```

Comment:
dodaj to jako zdjęcie w html i pozycja absolutna z with- heigth na 100% i object-fit: cover; lewo/prawo 0